src/GeneratorV2/Process/Overloaders.cs

```diff
@@ -2,23 +2,207 @@
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
+using System.Linq.Expressions;
 using System.Text.RegularExpressions;
 using GeneratorV2.Data;
 using GeneratorV2.Writing;
 
 namespace GeneratorV2.Process
 {
+    public sealed class VectorOverloader : IOverloader
+    {
+        private static readonly Regex VectorNameMatch = new Regex("([1-4])([f|d|h|i])v", RegexOptions.Compiled);
```

Comment:
Should the regex check to see that the `v` is the end of the line? Are there cases where we want the regex to match while it's in the middle of the name?
It would become something like
```regex
([1-4])([f|d|h|i])v$
```

Reply:
Yeah there are cases where the v comes in the middle of the name. These are also geenerally the methods with more than one vector in them. But just generally it is true for all methods with a vendor name postfixed aswell.

One example of this is `glNormal3fVertex3fSUN` but not sure if its only non extension methods. It seems like its only extension methods.